examples/digital-signature/Cargo.toml

```
@@ -14,7 +14,8 @@ tracing = { version = "0.1", features = ["log"] }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 
 [dev-dependencies]
-test-log = { version = "0.2", default-features = false, features = ["trace"] }
+env_logger = "0.10"
+test-log = { version = "0.2", features = ["trace", "log"] }
```

The default already contains `log`

```suggestion
test-log = { version = "0.2", features = ["trace"] }
```